inception-imls-dl4j/src/test/java/de/tudarmstadt/ukp/inception/recommendation/imls/dl4j/pos/DL4JSequenceRecommenderTest.java

```diff
@@ -333,6 +333,29 @@ public void thatNerEvaluationWorks() throws Exception
         assertThat(score).isBetween(0.0, 1.0);
     }
 
+    @Test
+    public void thatNerPredictionProducesNonPaddingLabels() throws Exception
```

Comment:
How about integrating the assert that no `NO_LABEL` predictions are produced into the other tests?